bundles/org.openhab.binding.upnpcontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -257,13 +289,21 @@
 		<item-type>String</item-type>
 		<label>Current Media Id</label>
 		<description>Current id of media entry or container, option list to browse hierarchy</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>MediaControl</tag>
+		</tags>
 	</channel-type>
 	<channel-type id="search">
 		<item-type>String</item-type>
 		<label>Search Criteria</label>
 		<description>Search criteria for searching the directory. Search criteria are defined in UPnP search criteria format.
 			Examples: dc:title contains "song", dc:creator contains "SpringSteen", unp:class = "object.item.audioItem",
 			upnp:album contains "Born in"</description>
+		<tags>
+			<tag>Status</tag>
```

Comment:
```suggestion
			<tag>Control</tag>
			<tag>Channel</tag>
```

Reply:
I will handle this in https://github.com/openhab/openhab-addons/pull/18544